bokeh/embed.py

```diff
@@ -59,7 +59,7 @@ def _wrap_in_script_tag(js):
     return SCRIPT_TAG.render(js_code=js)
 
 @contextmanager
-def _ModelInDocument(models, apply_theme=None):
+def _ModelInDocument(models, apply_theme=None, validate=True):
```

Comment:
because of performance concerns, it'd be nice to have the validation be configurable. I'll add a `validate` kwarg to all of the bokeh.io and bokeh.embed outputs